app.py

```diff
@@ -12,13 +18,13 @@
 logger = getLogger()
 
 
-def scrap_vessels_with_marine_traffic():
+def scrap_vessels_with_marine_traffic() -> None:
```

Comment:
Note sure why there are type annotations here? This looks like something which should be done in the context of #14

Reply:
Those return types have been introduced to respect linting rules.
Let's solve the #14 issue in another PR.